src/pages/trader-tools/pip-calculator/_pip-calculator.tsx

```diff
@@ -78,11 +79,56 @@ const PipCalculator = () => {
     const data = useStaticQuery(query)
 
     const [tab, setTab] = useState('Synthetic')
+    const [activeSymbols, setActiveSymbols] = useState([])
```

Comment:
can you please check this out?
https://github.com/binary-com/deriv-com/pull/3679/files#diff-3400a50adf53be0202182f57fb9249092025b0b23e267bee4b1b749696616a5a:~:text=const%20LiveMarketTable%20%3D,)%20%3D%3E%20%7B 
if you use `map` instead of `array` most of your operations will be easier, this is just a suggestion :pray: